app/helpers/admin/country_helper.rb

```diff
@@ -8,4 +8,10 @@ def en_ar_country(country)
 
     ("(#{en_country}) #{ar_country}").html_safe
   end
+
+  def country_options_for_select
+    Sponsor.distinct.pluck(:country).each_with_object({}) do |c, h|
```

Comment:
this is used to populate `form select` field with a list of countries. We don't want to show any duplication there.